scripts/import/import_hgmd.pl

```diff
@@ -81,7 +87,8 @@ sub add_variation {
   });
 
   my $insert_v_sth = $dbh->prepare(qq{
-    INSERT IGNORE INTO variation (name,source_id) VALUES (?,?);
+    INSERT IGNORE INTO variation (name,source_id,evidence_attribs, display)
```

Comment:
Remove space after comma to be consistent.

Reply:
Thanks for checking @at7, have updated it in the next commit.